packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1508,7 +1508,7 @@ export default abstract class SqlIntegration
             )}' AND'${formatDate(end, "yyyyMMdd")}'`,
           getAdditionalEvents: () => [],
           getMetricWhereClause: (eventName: string) =>
-            `WHERE event_name = '${eventName}'`,
+            `WHERE _TABLE_SUFFIX BETWEEN '{{date startDateISO "yyyyMMdd"}}' AND '{{date endDateISO "yyyyMMdd"}}' AND event_name = '${eventName}'`,
```

Comment:
I have updated this in: https://github.com/growthbook/growthbook/pull/1673 to get the intraday tables as well.  So if I land that first then you will have a merge conflict.  Or you can copy the code from that change.